pandas/core/indexes/interval.py

```diff
@@ -405,10 +405,6 @@ def values(self):
         """
         return self._data
 
-    @cache_readonly
-    def _values(self):
-        return self._data
```

Comment:
this is already defined in the base class as such

Reply:
this reminds me: typing.final i think is introduced in py38.  we could make our own shim in pd._typing for py<38 and start using it, might help avoid situations like this